roles/pulp_api/tasks/main.yml

```diff
@@ -62,17 +62,30 @@
   when:
     - (ansible_version.major > 2) or (ansible_version.major == 2 and ansible_version.minor >= 9)
 
+- name: Get database config
+  command: "cat {{ __pulp_db_fields_key_path }}"
+  delegate_to: "{{ __pulp_real_sole_database_config_host }}"
+  register: pulp_db_fields_key
+  changed_when: false
+  become: true
+
+- name: Check for existing Pulp Database Encryption Key
+  stat:
+    path: '{{ __pulp_db_fields_key_path }}'
+    get_attributes: false
+    get_checksum: false
+    get_mime: false
+  register: __pulp_db_fields_key_file
+  become: true
+
 - name: Import Pulp Database Encryption Key
```

Comment:
We already have an import key functionality that we tell the users they can use via variables.

I think a better name would be like "copy Pulp Database Encryption Key from pulp_database_config node"